app/backend/resources/asset-schema.edn

```diff
@@ -258,1388 +266,1412 @@
+  ["Bearing"
+   "Sildeehituselt tugisambale koormust üle kandev element."],
   :ctype/parent ":ctype/bridgesupport"}
  {:db/id ":ctype/expansionjoint",
   :db/ident :ctype/expansionjoint,
-  :db/doc "No",
   :asset-schema/type :asset-schema.type/ctype,
-  :asset-schema/label ["Paisumisvuuk" "Expansion joint"],
+  :asset-schema/label [true "Paisumisvuuk"],
+  :asset-schema/description ["Expansion joint" ""],
   :ctype/parent ":ctype/bridgesupport"}
  {:db/id ":ctype/bearinggirder",
   :db/ident :ctype/bearinggirder,
-  :db/doc "No",
   :asset-schema/type :asset-schema.type/ctype,
-  :asset-schema/label ["Riigel" "Bearing girder"],
+  :asset-schema/label [true "Riigel"],
+  :asset-schema/description ["Bearing girder" ""],
   :ctype/parent ":ctype/bridgesupport"}
  {:db/id ":ctype/supportwall",
   :db/ident :ctype/supportwall,
-  :db/doc "No",
   :asset-schema/type :asset-schema.type/ctype,
-  :asset-schema/label ["Tugisein" "Support wall"],
+  :asset-schema/label [true "Tugisein"],
+  :asset-schema/description ["Support wall" ""],
   :ctype/parent ":ctype/bridgesupport"}
  {:db/id ":ctype/wingwall",
   :db/ident :ctype/wingwall,
-  :db/doc "No",
   :asset-schema/type :asset-schema.type/ctype,
-  :asset-schema/label ["Samba tiibsein" "Wing wall"],
+  :asset-schema/label [true "Samba tiibsein"],
+  :asset-schema/description ["Wing wall" ""],
   :ctype/parent ":ctype/bridgesupport"}
  {:db/id ":ctype/icebreaker",
   :db/ident :ctype/icebreaker,
-  :db/doc "No",
   :asset-schema/type :asset-schema.type/ctype,
-  :asset-schema/label ["Jäämurdja" "Ice breaker"],
+  :asset-schema/label [true "Jäämurdja"],
+  :asset-schema/description ["Ice breaker" ""],
   :ctype/parent ":ctype/bridgesupport"}
  {:db/id ":ctype/scourprotection",
   :db/ident :ctype/scourprotection,
-  :db/doc "No",
   :asset-schema/type :asset-schema.type/ctype,
-  :asset-schema/label ["Erosioonitõke" "Scour protection"],
+  :asset-schema/label [true "Erosioonitõke"],
+  :asset-schema/description ["Scour protection" ""],
   :ctype/parent ":ctype/bridgesupport"}
  {:db/id ":ctype/slopeprotection",
   :db/ident :ctype/slopeprotection,
-  :db/doc "No",
   :asset-schema/type :asset-schema.type/ctype,
-  :asset-schema/label ["Nõlvakindlustus" "Slope protection"],
+  :asset-schema/label [true "Nõlvakindlustus"],
+  :asset-schema/description ["Slope protection" ""],
   :ctype/parent ":ctype/bridgesupport"}
  {:db/id ":ctype/stair",
   :db/ident :ctype/stair,
-  :db/doc "No",
   :asset-schema/type :asset-schema.type/ctype,
-  :asset-schema/label ["Trepp" "Stair"],
+  :asset-schema/label [true "Trepp"],
+  :asset-schema/description ["Stair" ""],
   :ctype/parent ":ctype/bridgesupport"}
  {:db/id ":ctype/expansionjointsupportbeam",
   :db/ident :ctype/expansionjointsupportbeam,
-  :db/doc "No",
   :asset-schema/type :asset-schema.type/ctype,
-  :asset-schema/label
-  ["Paisumisvuugi tugipruss" "Expansion joint support "],
+  :asset-schema/label [true "Paisumisvuugi tugipruss"],
```

Comment:
Can't have true in place of the label as it's defined as tuple fo strings. ["string1" "string2"]

Reply:
![image](https://user-images.githubusercontent.com/24251864/110787128-a8d13700-8275-11eb-882b-d1c6b41809ae.png)
aa okey so it's because Raul has added column C. 
I'll try to work a round it.